src/components/stack-chart/index.js

```diff
@@ -71,7 +71,7 @@ class StackChartGraph extends React.PureComponent<Props> {
       processDetails,
     } = this.props;
 
-    const maxViewportHeight = maxStackDepth * STACK_FRAME_HEIGHT;
+    const maxViewportHeight = (maxStackDepth + 1) * STACK_FRAME_HEIGHT;
```

Comment:
one question is: is maxStackDepth itself right ?  Is it because the depth is 0-based ?
If yes, I believe we should increment 1 in the reducer -- along with a comment explaining this :)

Reply:
First off, just a few moments before you commented I found out that this change messes up the canvas hit test when hovering around the top edge of the canvas, so this change shouldn't be merged. Sorry about that.

Yes, I think `maxStackDepth` is correct. If I'm not mistaken, a tree consisting of only a root has height 0 and that node has depth 0. And the max depth of that tree is 0. If we increase it by one in the reducer shouldn't we find a new name for it then?

Reply:
Unless we just define `maxDepth` to be the highest depth in the tree plus one, and explain it with a comment as you said.
I'll look into it, and fix hovering.

Reply:
Sorry, I'm thinking and working with call node trees with the flame graph. So that's why I started talking about trees even though this is a stack. So you can ignore what I said and I'll be back with a better solution :-)